src/Blockcore/Broadcasters/BroadcasterBase.cs

```diff
@@ -36,6 +36,10 @@ public void Init(ClientEventBroadcasterSettings broadcasterSettings)
         {
             this.log.LogDebug($"Initialising Web Socket Broadcaster {this.GetType().Name}");
 
+            // Why is this fetching form the wallet every 5 sec?
```

Comment:
This is probably how it originally was in the source code, someone else made the initial broadcaster. This should hopefully have a way to check "is there any listeners" check before doing any processing.

Reply:
ahh good you spotted that, I will try to see if I can check for listeners